### ISSUE TITLE
[enhancement] break out ARA results to volume and nginx proxy

### ISSUE THREAD

**Issue by leifmadsen** [redhat-nfvpe/ansible-cira#131]
Currently, logs are recovered from the jenkins slave and SCP'd back to the `/srv/static/` directory on the jenkins master. These files are then exposed via an NGINX proxy located on the master which performs redirects using a hostname and path.

Moving things forward in the Docker realm, we should actually create a volume in `docker-compose.yml` to expose the `/srv/static/` path into the volume. Then create an NGINX container that `volumes_from` jenkins master.

We can then also run ARA behind the NGINX proxy, and provide it access to the same volume, which ideally should permit loading the full web interface, loading the job data in to the sqlite proxy. That is likely a future enhancement and not part of the scope of this change.

**Comment by leifmadsen** [comment]
@yrobla, @dougbtv  I think this approach will work... ?

![image](https://cloud.githubusercontent.com/assets/768751/24278331/95b29308-1019-11e7-83bf-e90881c3bea4.png)

**Comment by dougbtv** [comment]
Just an FYI that I really like it. I like it busted out into a volume, and I like nginx to serve that content.

**Comment by leifmadsen** [comment]
One potential question will be around how this affects our OpenStack deployments, and whether we're still going to support that approach. A lot of these Docker changes could start to affect how we deploy in other topologies.

**Comment by yrobla** [comment]
Yes, please try to make it compatible with the OpenStack approach, and also keep backwards functionality, to avoid redeploying existing environments.

**Comment by leifmadsen** [comment]
OK so things aren't really broken out the way I originally had it envisioned in my head. Of course I think that would be the more "correct" way of doing it, but we have a lot of rules we break already in the name of science^Wspeed :)

Basically all I'm going to propose for now is actually moving trafik to point at NGINX running on the Jenkins Master as a proxy. This is the same way it works in the VM deployment, and requires the least amount of changes on our Docker deployment.
